libcxx/test/std/input.output/filesystems/class.file_status/file_status.status.eq.ops.cpp

```diff
@@ -22,26 +22,26 @@
 
 void test() {
```

Comment:
This test isn't running because it's not recognized as a test. I've created #73444 to fix it properly.

Reply:
Thanks - reverted here!